src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -316,7 +316,7 @@ private void assertThatUserCanClickPostAndGoToPostDetailsPage() {
   }
 
   private void cleanUp(PostEntity.Data post) {
-    DiscussionsOperations.using(User.STAFF, driver).deletePost(post, siteId);
+    DiscussionsClient.using(User.STAFF, driver).deletePost(post, siteId);
   }
 
   private void assertPostWithWeirdCharactersDisplayedOnPostsListPage(String wiki) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "assertPostWithWeirdCharactersDisplayedOnPostsListPage" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)